### PR TITLE
solution

### PR DESCRIPTION
**Pull request by 1stIndominus**
[DEMO LINK](https://1stIndominus.github.io/js_notification_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+  div.style.right = posRight + 'px';
+  div.style.top = posTop + 'px';
+
+  div.style.boxSizing = 'content-box';
+  h2.className = 'title';
+  h2.textContent = title;
+  p.textContent = description;
+
+  div.classList = 'notification';
+  div.classList.add(type);
```
Comment (polosanya) [inline src/scripts/main.js:19]:
```suggestion
  div.classList.add('notification', type);
```

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
```
Comment (polosanya) [inline src/scripts/main.js:4]:
```suggestion
  const notification = document.createElement('div');
```
Don't name variables the same as tag. Try to give them more descriptive names

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+  const body = document.querySelector('body');
+
+  div.style.position = 'absolute';
+  div.style.right = posRight + 'px';
+  div.style.top = posTop + 'px';
+
+  div.style.boxSizing = 'content-box';
```
Comment (polosanya) [inline src/scripts/main.js:13]: Do you really need to add it? Same with position absolute.

**Review by polosanya — CHANGES_REQUESTED** [review]
Let's improve your code a bit

File: src/scripts/main.js (hunk 3 above)
Comment (1stIndominus) [reply src/scripts/main.js:13]: I don't need position only but without content-box it will be broken

**Review by polosanya — APPROVED** [review]

**Pull request closed without merging**